toolchain/binutils.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+
+import os
+from avocado import Test
+from avocado import main
+from avocado.utils import archive
+from avocado.utils import build
+from avocado.utils import process
+from avocado.utils.software_manager import SoftwareManager
+
+backend = SoftwareManager()
+
+
+class Binutils(Test):
+    """
+    This testcase make use of unit testsuite provided by the
+    source package, performs functional test for all binary tools
+    source file is downloaded and compiled.
+    """
+
+    def setUp(self):
+        '''
+        Install required tools and resolve dependencies
+        '''
+        backend.install('rpmbuild')
+        backend.install('elfutils')
+        backend.install('build')
+        backend.install('autoconf')
+        backend.install('automake')
+        backend.install('binutils-devel')
+        backend.install('dejagnu')
+        backend.install('libtool')
```

Comment:
Also the `backend.install` does not raise exception when it fails to install the package. Would you please add `self.skip` with the list of packages which failed to install? If you expect some install failures (as package names varies over distros) you might either add `res = backend.install("glibc-static") or backend.install("glibc-devel-static")` and then check for res.

Alternatively if you don't see those as a big problem, please add at least `self.warn` listing the packages which failed to install. Usually the binutils tests fail due to incorrect environment.